.changeset/wicked-birds-push.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
More changeset details, also should probably be a minor.

Initially I thought this would be breaking, but I see now that you exported an instance of the class as well, so people can continue to use it as before.

Reply:
Done https://github.com/pingdotgg/uploadthing/commit/6ece5176f56e4700b8042ed3e75caad94b1b30f3